File: Java/Power_of_Four.java
void main(){
    Scanner scanner = new Scanner(System.in);
    int a = scanner.nextInt();

    if (a <= 0){
        System.out.println("4 un quvveti deyil");
        scanner.close();
        return;
    }

    while (a % 4 == 0) {
        a /= 4;
    }

    if(a == 1){
        System.out.println("4 un quvvetidir");
    }
    else{
        System.out.println("4 un quvveti deyil");
    }

    scanner.close();
}

File: Python/Power_of_Four.py
def power_of_four():
    a = int(input())

    if a <= 0:
        print("4 un quvveti deyil")
        return

    while a % 4 == 0:
        a //= 4

    if a == 1:
        print("4 un quvvetidir")
    else:
        print("4 un quvveti deyil")

if __name__ == "__main__":
    power_of_four()
